src/encoded/static/components/facets/defaults.js

```diff
@@ -154,8 +154,8 @@ export const DefaultExistsBinaryFacet = ({
     // So set YES-radio button to the facet-no count and set the NO-radio button to both the combined
     // facet-yes and facet-no counts
     const terms = [
-        { key: 'yes', doc_count: facetNoTerm.doc_count },
-        { key: 'no', doc_count: facetYesTerm.doc_count + facetNoTerm.doc_count },
+        { key: 'yes', doc_count: facetNoTerm?.doc_count },
+        { key: 'no', doc_count: facetNoTerm && facetYesTerm && (facetYesTerm.doc_count + facetNoTerm.doc_count) },
```

Comment:
`facetNoTerm && facetYesTerm && (facetYesTerm.doc_count + facetNoTerm.doc_count)`

Evaluates to a bool. Is that your intent?